src/v/security/audit/schemas/utils.cc

```diff
@@ -206,6 +206,20 @@ actor actor_from_request_auth_result(
     return {.authorizations = std::move(auths), .user = std::move(u)};
 }
 
+actor actor_from_user_string(const ss::sstring& user) {
+    struct user u = {
+      .name = user.empty() ? "{{anonymous}}" : user,
+      .type_id = user.empty() ? user::type::unknown : user::type::user,
+    };
+    std::vector<authorization_result> auths{
+      {.decision = "authorized",
+       .policy = policy{
+         .desc = user.empty() ? "Auth Disabled" : "",
+         .name = "Schema Registry httpd authorizer"}}};
```

Comment:
Should this belong in Schema Registry, or have the name be a parameter?

Reply:
Provided via the service name